### PR TITLE
refactor scraper config

### PR DESCRIPTION
**Pull request by yingjieMiao**
@eishay @raymondng 

factor out a `ScraperSchedulerConfig`. With current code, this HAS to be accessible by both shoebox and scraper. (shoebox uses this to schedule scraping. Scraper uses it to set nextScrapeTime, etc.)

I am not sure this really simplifies things. Moving ScrapeInfo table to Scraper is the right solution.

### CODE REVIEW

**Comment by eishay** [comment]
Thanks @yingjieMiao !
I agree, moving the table should be the next step.

File: kifi-backend/modules/common/app/com/keepit/scraper/ScraperSchedulerConfig.scala [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+  checkOverdueCountFrequency: Int,      // minutes
+  pendingOverdueThreshold: Int,         // minutes
+  overdueCountThreshold: Int,
+  intervalConfig: ScraperIntervalConfig
+) {
+  private[this] val rnd = new Random
+  def randomDelay(): Int = rnd.nextInt(intervalConfig.maxRandomDelay)     // seconds
+}
+
+trait ScrapeSchedulerConfigModule extends ScalaModule {
```
Comment (raymondng) [inline kifi-backend/modules/common/app/com/keepit/scraper/ScraperSchedulerConfig.scala:32]: Minor: `ScraperScheduler` at this point, is a bit of a misnomer -- it's really a Scraper health-check monitor.

**Comment by raymondng** [comment]
Overall lgtm. Thanks!

**Comment by yingjieMiao** [comment]
merging this. will babysit the deploy. hope no misconfig.

**Comment by raymondng** [comment]
You can check test-demand-1 to ensure that at least scraper deploys
correctly.

**Comment by yingjieMiao** [comment]
thanks!

**Pull request merged**